python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -1158,11 +1158,15 @@ def makeCredChecker(self, credObj):
         def credChecker():
             log.debug("Checking %s." % getName(credObj))
             try:
-                s = credObj.renew()
-            except Exception as msg:
+                credObj.renew()
+            except CredentialRenewalError:
```

Comment:
I was expecting to find this in `Ganga/Core/exceptions` with the rest of the definitions, is this class ultimately going to be placed here?